bin/terraform

```diff
@@ -6,16 +6,17 @@ source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
 
 args=( "$@" )
 
-args+=("-var-file=/config/config.tfvars")
-
-if [[ "$1" == "init" ]] ; then
-    args+=(
-        "-backend-config=bucket=${RICHIE_SITE}-terraform"
-        "-backend-config=key=${RICHIE_SITE}.tfstate"
-        "-backend-config=dynamodb_table=${RICHIE_SITE}_terraform_state_locks"
-    )
-fi
+case "$1" in
+    apply|destroy|import|plan)
+        args[0]="${args[0]} -var-file=/config/config.tfvars"
+        ;;
+    init)
+        args[0]="${args[0]} -backend-config=key=${RICHIE_SITE}.tfstate"
+        ;;
+esac
 
 # Run Terraform commands in the Hashicorp docker container, passing our
 # environment variables
-DOCKER_USER="$(id -u):$(id -g)" docker-compose run --rm terraform "${args[@]}"
+# shellcheck disable=SC2068
+DOCKER_USER="$(id -u):$(id -g)" \
+    docker-compose run --rm terraform ${args[@]}
```

Comment:
@jmaupetit shellcheck is not happy with this but it doesn't work if I do:
```suggestion
    docker-compose run --rm terraform "${args[@]}"
```
So I ignored it but if you have a better solution don't hesitate!

Reply:
I think you are right on this: you explicitly want to split array elements or else docker-compose cannot correctly interpret your command.

Reply:
:+1: